spec/models/recipe_spec.rb

```diff
@@ -12,4 +12,16 @@
     it {should have_many :recipe_ingredients}
     it {should have_many(:ingredients).through(:recipe_ingredients)}
   end
+
+  describe "#instance methods" do
+    it "#total_cost" do
+      ingredient1 = Ingredient.create!(name: "Sugar", cost: 2)
```

Comment:
Nice work testing this model method! I think you are just missing the model test for the alphabetical order of ingredients.

Reply:
The alpha order is an extension though so missing that test won't affect your grade.